examples/src/components/PivotTableDrillingExample.jsx

```diff
@@ -0,0 +1,262 @@
+        label: 'Measures',
+        key: 'measures',
+        bucketProps: {
+            measures
+        }
+    },
+    columnAttributes: {
+        label: 'Column attributes',
+        key: 'columnAttributes',
+        bucketProps: {
+            columns
+        }
+    },
+    rowAttributes: {
+        label: 'Row attributes',
+        key: 'rowAttributes',
+        bucketProps: {
+            rows
+        }
+    },
+    columnAndRowAttributes: {
+        label: 'Column and row attributes',
+        key: 'columnAndRowAttributes',
+        bucketProps: {
+            columns,
+            rows
+        }
+    },
+    measuresColumnAndRowAttributes: {
+        label: 'Measures, column and row attributes',
+        key: 'measuresColumnAndRowAttributes',
+        bucketProps: {
+            measures,
+            columns,
+            rows
+        }
+    },
+    measuresAndColumnAttributes: {
+        label: 'Measures and column attributes',
+        key: 'measuresAndColumnAttributes',
+        bucketProps: {
+            measures,
+            columns
+        }
+    },
+    measuresAndRowAttributes: {
+        label: 'Measures and row attributes',
+        key: 'measuresAndRowAttributes',
+        bucketProps: {
+            measures,
+            rows
+        }
+    }
+};
+
+const drillingPresets = {
+    measure: {
+        label: 'Measure Franchise Fees',
+        key: 'measure',
+        drillableItem: {
+            identifier: franchiseFeesIdentifier
+        }
+    },
+    attributeMonth: {
+        label: 'Attribute Month',
+        key: 'attributeMonth',
+        drillableItem: {
+            identifier: monthDateIdentifier
+        }
+    },
+    attributeYear: {
+        label: 'Attribute Year',
+        key: 'attributeYear',
+        drillableItem: {
+            identifier: yearDateIdentifier
+        }
+    },
+    attributeLocationState: {
+        label: 'Attribute Location state',
+        key: 'attributeLocationState',
+        drillableItem: {
+            identifier: locationStateDisplayFormIdentifier
+        }
+    },
+    attributeMenuCategory: {
+        label: 'Attribute Menu category',
+        key: 'attributeMenuCategory',
+        drillableItem: {
+            identifier: menuCategoryAttributeDFIdentifier
+        }
+    },
+    attributeValueCalifornia: {
+        label: 'Attribute value California',
+        key: 'attributeValueCalifornia',
+        drillableItem: {
+            uri: locationStateAttributeCaliforniaUri
+        }
+    },
+    attributeValueJanuary: {
+        label: 'Attribute value January',
+        key: 'attributeValueJanuary',
+        drillableItem: {
+            uri: monthDateIdentifierJanuary
+        }
+    }
+};
+
+export class PivotTableDrillingExample extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            bucketPresetKey: 'measuresColumnAndRowAttributes',
+            drillEvent: null,
+            drillingPresetKeys: {
```

Comment:
This looks to me like a Set (https://facebook.github.io/immutable-js/docs/#/Collection.Set)

Reply:
I stand by object literal here.